### PR TITLE
FLUID-5937: Added some tweaks to mobile css

### PR DESCRIPTION
**Pull request by jhung**
- tightened up margins and padding
- changes word wrap behaviour in mobile
- fixed bug where UIO enlarged links would cause the hide-show button to layout improperly.

The mobile experience can be improved overall, but will require additional work which isn't quick (i.e. changing the way the sidebar behaves and the hamburger menu).

You can see this version of the guides here: https://jhung.github.io/guide.inclusivedesign.ca/

### CODE REVIEW

File: src/documents/css/idg.css.styl [hunk 1]
```diff
@@ -355,11 +355,44 @@ setActiveCategory(inputColor) {
 
 /* Styles specific to small screens. Using Foundation 5 value for small screen width */
 @media only screen and (max-width: 40em) {
-    .docs-template-articleBody h1 {
-        font-size: 1.7rem;
+
+    * {
+        word-wrap: break-word;
```
Comment (jobara) [inline src/documents/css/idg.css.styl]: Do we really want text that looks like this?

screen shot 2016-08-03 at 12 44 48 pm

Comment (jobara) [inline src/documents/css/idg.css.styl]: If we do need this, can we also add hyphens, which would hopefully make things look better at least in the browsers that support it.

https://css-tricks.com/almanac/properties/h/hyphenate/

**Comment by jhung** [comment]
@jobara I've updated the pull. Added hyphenation and the word break is more specific - now only applies to the sidebar and the article h1.

**Comment by jhung** [comment]
I think for now this is okay. Hyphens don't show in Chrome and IE11, but the words still break - which is better than the current behaviour which is to overflow long words.

**Comment by jobara** [comment]
merged at edc5db1760a4a374f6e6e845ba4f013d7c71954d

**Pull request merged**